src/screens/admin/DocumentConfig.tsx

```
@@ -617,6 +667,150 @@ const DocumentConfig = () => {
+														</Text>
+													</FormLabel>
+													<Select
+														value={
+															doc.docQRContains ||
+															''
+														}
+														onChange={(e) =>
+															handleChange(
+																index,
+																'docQRContains',
+																e.target.value as DocumentConfig['docQRContains']
```

ESLint

[error] Insert `⏎↹↹↹↹↹↹↹↹↹↹↹↹↹↹↹↹↹`

(prettier/prettier)